### PR TITLE
add a new game state

### PR DESCRIPTION
**Pull request by tim-hub**
pre game start to logging

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+config.py
```
Comment (toshima) [inline .gitignore]: Thanks for the PR! The change to generals.py is good, but please could you undo the change to .gitignore?

**Review by toshima — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (tim-hub) [reply .gitignore]: yes

Comment (tim-hub) [reply .gitignore]: yes did it. is that ok?

**Review by tim-hub — COMMENTED** [review]
I removed it

**Comment by toshima** [comment]
I wanted the line you added to .gitignore to be removed, not the whole file to be deleted.

If you could add back the .gitignore file without the line you added before (config.py), I'd be happy to merge :)

**Pull request closed without merging**